src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -16,13 +16,17 @@ const propTypes = {
     /** Key-value pairs of user logins and whether or not they are typing. Keys are logins. */
     userTypingStatuses: PropTypes.objectOf(PropTypes.bool),
 
+    /** Optional styles for container element that will override the default styling for the report typing indicator */
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
+
```

Comment:
We have this prop used on this line: https://github.com/Expensify/App/blob/90b5292c89ff4516d3f4902d728dcabb67ce3786/src/pages/home/report/ReportTypingIndicator.js#L81 

in the component but we have not defined a default prop so if not present and trying to spread it we get an error. Simple fix. It is hard to reproduce however.